uploadNeuroDB/minc_insertion.pl

```diff
@@ -160,8 +163,14 @@
     exit 3; 
 }
 
+if ( ($tarchive && $upload_id) || (!$tarchive && !$upload_id) ) {
```

Comment:
Use the `xor` operator here. It tests that one of the two values is set but not both. So the above test could be written as:

`if (!($tarchive xor $upload_id))`

Reply:
ouhh, nice!